modules/mcuboot/CMakeLists.txt

```
@@ -16,15 +16,16 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
     # SIGNED_HEX_FILE_OUT - (required) Signed hex output image
     # DEPENDS - (optional) One or more dependencies for signing the hex image
 
-    set(oneValueArgs SIGNED_BIN_FILE_IN SIGNED_HEX_FILE_NAME_PREFIX START_ADDRESS_OFFSET SIGNED_HEX_FILE_OUT)
+    set(oneValueArgs SIGNED_BIN_FILE_IN SIGNED_HEX_FILE_NAME_PREFIX SLOT_SIZE START_ADDRESS_OFFSET SIGNED_HEX_FILE_OUT)
     set(multiValueArgs DEPENDS)
     cmake_parse_arguments(SIGN_ARG "" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
     if (NOT DEFINED SIGN_ARG_SIGNED_BIN_FILE_IN
         OR NOT DEFINED SIGN_ARG_SIGNED_HEX_FILE_OUT
         OR NOT DEFINED SIGN_ARG_SIGNED_HEX_FILE_NAME_PREFIX
-        OR NOT DEFINED SIGN_ARG_START_ADDRESS_OFFSET)
-      message(FATAL_ERROR "Missing parameter, required: SIGNED_BIN_FILE_IN SIGNED_HEX_FILE_NAME_PREFIX START_ADDRESS_OFFSET SIGNED_HEX_FILE_OUT")
+        OR NOT DEFINED SIGN_ARG_START_ADDRESS_OFFSET
+        OR NOT DEFINED SIGN_ARG_SLOT_SIZE)
+      message(FATAL_ERROR "Missing parameter, required: SIGNED_BIN_FILE_IN SIGNED_HEX_FILE_NAME_PREFIX START_ADDRESS_OFFSET SIGNED_HEX_FILE_OUT SLOT_SIZE")
     endif()
```

Consider changing this code block to:
https://github.com/nrfconnect/sdk-nrf/blob/d34214d69056671a3d9184332c139f98edbebff0/cmake/extensions.cmake#L39
```suggestion
  check_arguments_required_all(sign SIGN_ARG ${oneValueArgs})
```